lib/rubygems/commands/contents_command.rb

```diff
@@ -167,18 +167,16 @@ def show_files(files)
   end
 
   def spec_for(name)
-    spec = Gem::Specification.find_all_by_name(name, @version).last
-
-    return spec if spec
-
+    Gem::Specification.find_by_name(name, @version)
+  rescue Gem::MissingSpecError
     say "Unable to find gem '#{name}' in #{@path_kind}"
 
     if Gem.configuration.verbose
       say "\nDirectories searched:"
       @spec_dirs.sort.each { |dir| say dir }
     end
 
-    return nil
+    nil
```

Comment:
I prefer to explicitly call `return nil`,  just `nil` at the end seems weird

Reply:
I was just following https://github.com/rubocop-hq/ruby-style-guide#no-explicit-return and matching the way `true` is returned on L127 and L137 but I'm happy to remove this change.

Reply:
IMHO it's good practice to not touch lines which do not need to be touched.

Reply:
Neutral to this style change, but if we decide to enable that cop, it should be done as part of a separate PR and discussion.

Reply:
Agreed on leaving other lines untouched as a general rule -- in this case it was not to enforce the style cop by itself so much as it was to make this half of the block parallel the other half after the addition of the `rescue`. (This line itself wasn't touched but its block context did change.)

I've updated the PR though and this is no longer in scope.